frontend/src/pages/PropertyDetails.jsx

```
@@ -0,0 +1,403 @@
+                            )}
+                        </div>
+                    </div>
+                    
+                    {/* Booking Form */}
+                    {property.status === "available" ? (
+                    <div className="lg:col-span-1">
+                        <div className="bg-white shadow-md rounded-lg p-6 sticky top-8">
+                            <h2 className="text-2xl font-bold text-gray-800 mb-4">Book Now</h2>
+                            <div className="space-y-4">
+                                <div>
+                                    <label className="block text-gray-700 mb-2">Check In</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)